pkg/sql/row/row_converter.go

```diff
@@ -331,12 +332,18 @@ func NewDatumRowConverter(
 	}
 
 	var txCtx transform.ExprTransformContext
-	semaCtx := tree.MakeSemaContext()
 	relevantColumns := func(col catalog.Column) bool {
 		return col.HasDefault() || col.IsComputed()
 	}
+
+	// We take a copy of the semaCtx since this method is called by the parallel
+	// import workers.
+	semaCtxCopy := tree.MakeSemaContext()
```

Comment:
nit: we only need to make a new one if the arg is nil, so we could start with a `var` and do `make` in an else branch on the if that does the de-reference below. idle curiosity: how hard would it be to never pass a nil semaCtx to this method and make the callers call Make if needed?

Reply:
heh, I checked all the call sites and everyone already seems to. I'll add an assertion to prevent an NPE.

Reply:
/shrug if everyone always passes it non-nil, I think it is fine to just dereference it -- we don't typically nil-check all our args.